app/mailers/single_cell_mailer.rb

```diff
@@ -36,6 +36,15 @@ def notify_admin_upload_fail(study_file, error)
     end
   end
 
+  def notify_user_upload_fail(study_file, study, user)
```

Comment:
add a method comment 'notifies the user and cc'es dev team if appropriate for the environment' or somesuch

Reply:
4fb8abb